src/WebJobs.Extensions.Http/WebJobs.Extensions.Http.csproj

```diff
@@ -7,18 +7,23 @@
     <PackageId>Microsoft.Azure.WebJobs.Extensions.Http</PackageId>
     <Description>This package adds binding extensions for Http.</Description>
   </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="..\WebJobs.Extensions\Framework\Bindings\StreamValueBinder.cs" Link="Binding\StreamValueBinder.cs" />
```

Comment:
Linking to the extension project files we needed. Will create an issue to revisit this and move completely to the new binding model.

Reply:
Will this be an issue when other extensions also link these public types, resulting in duplicate type exports if a customer app pulls in Http and SendGrid extensions?

Reply:
Looking at this again, they'll become internal as we migrate the extensions. Once all the migrations are done, we can keep them in a shared project (or get rid of them)